pyserini/index/_base.py

```diff
@@ -150,6 +151,15 @@ def __init__(self, index_dir):
         self.object = JIndexReader()
         self.reader = self.object.getReader(JString(index_dir))
 
+    @classmethod
```

Comment:
Can we add docstrings to this and below?

Reply:
I add docstrings to `IndexReader` and `SimpleSearcher`. Ready for review.